Cargo.toml

```
@@ -1,6 +1,6 @@
 [package]
 name = "tracing-opentelemetry"
-version = "0.17.4"
+version = "0.19.0"
```

Let's update the released version in a separate PR to simplify the merge sequence

```suggestion
version = "0.18.0"
```